frontend/src/scenes/annotations/logic.js

```diff
@@ -7,7 +7,8 @@ export const annotationsTableLogic = kea({
         annotations: {
             __default: [],
             loadAnnotations: async () => {
-                const response = await api.get('api/annotation/?' + toParams({ order: '-updated_at' }))
+                const params = { order: '-updated_at', deleted: false }
```

Comment:
Given that we're now really deleting stuff, I think the `deleted` attribute is extra? Maybe we should include a migration to really delete all the soft-deleted annotations? We could use this as a test drive for doing the appropriate housekeeping in all models.

Reply:
That could be done yes